packages/api/tests/lib/writeRecords/test-write-execution.js

```diff
@@ -514,3 +547,27 @@ test('writeExecutionRecordFromApi() saves execution to Dynamo and RDS with same
   t.is(pgRecord.created_at.getTime(), dynamoRecord.createdAt);
   t.is(pgRecord.updated_at.getTime(), dynamoRecord.updatedAt);
 });
+
+test.serial('writeExecutionRecordFromMessage() calls publishExecutionSnsMessage and successfully publishes an SNS message', async (t) => {
+  const {
+    cumulusMessage,
+    executionArn,
+    executionModel,
+    knex,
+    QueueUrl,
+  } = t.context;
+
+  await writeExecutionRecordFromMessage({ cumulusMessage, knex });
+
+  const { Messages } = await sqs().receiveMessage({ QueueUrl, WaitTimeSeconds: 10 }).promise();
+
+  t.is(Messages.length, 1);
+
+  const snsMessage = JSON.parse(Messages[0].Body);
+  const executionRecord = JSON.parse(snsMessage.Message);
+  const dynamoRecord = await executionModel.get({ arn: executionArn });
+
+  t.is(executionRecord.arn, executionArn);
+  t.is(executionRecord.status, cumulusMessage.meta.status);
+  t.deepEqual(omit(executionRecord, 'timestamp'), omit(dynamoRecord, 'timestamp'));
```

Comment:
There should be no need to `omit` when comparing against the Dynamo record

Reply:
For posterity, we talked about this offline and timestamp should be the same across PG and dynamo records, and we need to update how this is implemented